examples/src/main/java/io/kubernetes/client/examples/Example.java

```diff
@@ -9,36 +9,230 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-*/
+ */
 package io.kubernetes.client.examples;
 
 import io.kubernetes.client.ApiClient;
 import io.kubernetes.client.ApiException;
 import io.kubernetes.client.Configuration;
 import io.kubernetes.client.apis.CoreV1Api;
-import io.kubernetes.client.models.V1Pod;
+import io.kubernetes.client.apis.ExtensionsV1beta1Api;
+import io.kubernetes.client.models.ExtensionsV1beta1Deployment;
+import io.kubernetes.client.models.ExtensionsV1beta1DeploymentList;
+import io.kubernetes.client.models.ExtensionsV1beta1DeploymentSpec;
+import io.kubernetes.client.models.V1NamespaceList;
 import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.models.V1ServiceList;
 import io.kubernetes.client.util.Config;
 import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 /**
  * A simple example of how to use the Java API
  *
- * <p>Easiest way to run this: mvn exec:java
+ * <p>
+ * Easiest way to run this: mvn exec:java
  * -Dexec.mainClass="io.kubernetes.client.examples.Example"
  *
- * <p>From inside $REPO_DIR/examples
+ * <p>
+ * From inside $REPO_DIR/examples
  */
 public class Example {
-  public static void main(String[] args) throws IOException, ApiException {
-    ApiClient client = Config.defaultClient();
-    Configuration.setDefaultApiClient(client);
 
-    CoreV1Api api = new CoreV1Api();
-    V1PodList list =
-        api.listPodForAllNamespaces(null, null, null, null, null, null, null, null, null);
-    for (V1Pod item : list.getItems()) {
-      System.out.println(item.getMetadata().getName());
+    private final CoreV1Api corev1Api;
```

Comment:
I don't think this needs to be declared as a class level variable.

Reply:
In the Constructor of Example() class, I initialized only one times for CoreV1Api instance as follows.
corev1Api = new CoreV1Api(client);

After that I reuse the instance valuable on following.
****************************************************************************
        V1NamespaceList listNamespace = corev1Api.listNamespace("true", null, null, Boolean.FALSE, null, 0, null, Integer.MAX_VALUE, Boolean.FALSE);

        V1PodList v1podList = corev1Api.listPodForAllNamespaces(null, null, null, null, null, null, null, null, null);

        V1PodList listNamespacedPod = corev1Api.listNamespacedPod(namespace, null, null, null, Boolean.FALSE, label, Integer.MAX_VALUE, null, TIME_OUT_VALUE, Boolean.FALSE);

        V1ServiceList listNamespacedService = corev1Api.listNamespacedService(DEFAULT_NAME_SPACE, null, null, null, Boolean.FALSE, null, Integer.MAX_VALUE, null, TIME_OUT_VALUE, Boolean.FALSE);

        extensionV1Api.setApiClient(corev1Api.getApiClient());
****************************************************************************

It may be OK to specify the "ApiClient client" instead of the "CoreV1Api".
ApiClient client = Config.defaultClient();

However I thought that "CoreV1Api" is better.